index.css

```diff
@@ -1,20 +1,27 @@
 @import '@navikt/ds-css' layer(aksel);
 
 html,
-        body {
-            display: flex;
-            flex-direction: column;
-            height: 100%;
-            margin: 0;
-        }
+body {
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    margin: 0;
+}
```

Comment:
Ikke noe du faktisk har endret på men mer en generell kommentar: vi burde unngå å style ting som er utenfor "scopet" til applikasjonen vår, dvs. `body` og `html` taggene samt `#veilarbpersonflate-root`-elementet. Dette er ting som veilarbpersonflatefs eier og det kan gi uforutsette konsekvenser om vi forsøker å override stylingen. Vi bør holde oss innenfor vår egen kontekst, som blir f.o.m. `main`-taggen og nedover i hierarkiet 👍

Reply:
Ser ut som det kommer herfra: [veilarbmaofs/public/index.html#L9-L26](https://github.com/navikt/veilarbmaofs/blob/master/public/index.html#L9-L26)!
Det forklarer `#main` og hvorfor `body` og `html` er stylet (for lokal kjøring), men det burde inlines sånn som i maofs 😁

Reply:
Hmm, vil tro det 🤔 
Men det enkleste, siden vi ikke vet hvordan man gjør ovennevnte, er vel å inline det i `index.html` siden den ikke er en del av det som eksporteres med NAVSPA

Reply:
Enig!